session/bootstrap.go

```diff
@@ -313,7 +313,7 @@ const (
 		ROWS_SELECTED bigint(64),
 		LAST_USED_AT timestamp,
 		LAST_UPDATED_AT timestamp,
-		PRIMARY KEY(TABLE_SCHEMA, TABLE_NAME, INDEX_NAME)
+		INDEX(TABLE_SCHEMA, TABLE_NAME, INDEX_NAME)
```

Comment:
Should we use `unique index` here?

Reply:
In consideration of performance and ease of implementation, I used `insert` instead of `update`, so this cannot be unique index here. Is there any hidden danger in doing this?

Reply:
There'll be multiple lines for an index and what we want is the one with the latest `LAST_UPDATE_AT`?

Reply:
Since we use `insert into ... on duplicate key update ...` to update records, can we add the unique attribute to this index now? @rebelice